pyproject.toml

```diff
@@ -216,3 +220,229 @@ check_untyped_defs = true
 # disallow_any_generics = true
 # no_implicit_reexport = true
 # disallow_untyped_defs = true
+
+
+# individual ignore error
+# we should strive to remove those
```

Comment:
Might be good to reference the issue tracking that effort: https://github.com/napari/napari/issues/2751